data/lib/src/network/model/local/document_dto.dart

```diff
@@ -30,27 +30,22 @@
 //  the Additional Terms applicable to LinShare software.
 //
 
+import 'package:data/src/network/model/converter/boolean_converter.dart';
+import 'package:data/src/network/model/converter/data_from_json_converter.dart';
+import 'package:data/src/network/model/converter/datetime_converter.dart';
+import 'package:data/src/util/attribute.dart';
 import 'package:domain/domain.dart';
-import 'package:domain/src/model/document/document_id.dart';
 import 'package:equatable/equatable.dart';
+import 'package:json_annotation/json_annotation.dart';
 import 'package:http_parser/http_parser.dart';
 
-class DocumentOffline extends Equatable {
-  final DocumentId documentId;
-  final String description;
-  final DateTime creationDate;
-  final DateTime modificationDate;
-  final DateTime expirationDate;
-  final bool ciphered;
-  final String name;
-  final int size;
-  final String sha256sum;
-  final bool hasThumbnail;
-  final int shared;
-  final MediaType mediaType;
-  final String localPath;
+part 'document_dto.g.dart';
 
-  DocumentOffline(
+@JsonSerializable()
+@DatetimeConverter()
+@BooleanConverter()
+class DocumentDto extends Equatable {
```

Comment:
we can not using the same model for network and for database. DocumentDTO from network don't care to `localPath`

Reply:
This is just an object for handling the database it has nothing to do with the network. Maybe my naming makes you misunderstood.